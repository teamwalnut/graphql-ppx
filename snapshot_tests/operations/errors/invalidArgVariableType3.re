/**
 * These are just mismatched types, the optional string should throw an error
 */
module MyQuery = [%graphql
  {|
  query invalidArgValues($arg: String!, $arg2: Int) {
    invalidArgValues(
        stringRequired: $arg
        string: $arg2
    )
  }
|}
];
