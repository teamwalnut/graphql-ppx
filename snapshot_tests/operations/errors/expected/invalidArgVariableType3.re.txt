
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/invalidArgVariableType3.re[0m:[2m9:17-21[0m

   7 [2mâ”†[0m   invalidArgValues(
   8 [2mâ”†[0m       stringRequired: $arg
   [1;31m9[0m [2mâ”†[0m       string: [1;31m$arg2[0m
  10 [2mâ”†[0m   )
  11 [2mâ”†[0m }

  Invalid argument. Expected "String" but received "Int".

